sections/combat.tex

```diff
@@ -12,6 +12,18 @@
 \end{itemize}
 Players are able to start multiple Combats during their Turn.
 
+\note{17}{
+  If your Town or Settlement is attacked by an enemy Hero and your Hero is not on that Field, you may immediately \textbf{pay 8} \svg{gold} to fight a defending Combat \textbf{using only your Units}.
+  You cannot use your Deck during that Combat as your Main Hero is not present.
+  Paying this Gold represents the cost of transporting the army there.
+
+  \vspace*{1em}
+
+  When a \pagelink{Secondary}{Secondary Hero} is attacked, they \textbf{may choose} to be \pagelink{Endcombat}{instantly defeated} instead of fighting a Combat, because this helps preserving the Units.
```

Comment:
```suggestion
  When a \pagelink{Secondary}{Secondary Hero} is attacked, they \textbf{may choose} to be \pagelink{Endcombat}{instantly defeated} instead of engaging in Combat, which helps preserve the Units.
```